README.md

```diff
@@ -51,6 +51,33 @@ MY_PROJ_DIR
 Whenever you update the dependencies declared in dependencies.yaml you will need to regenerate the
 contents of the `3rdparty` directory by re-running `$BAZEL_DEPS/gen_maven_deps.sh generate`.
 
+### Alternate outputs, external repo
+Bazel-deps can also prepare the outputs, not as a file tree but an external repo. With this one would refer to targets as
+`@third_party//foo:bar` rather than `//3rdparty/jvm/foo/bar`. This is useful if you do not want to check in generated code to your
```

Comment:
is this `third_party//jvm/`? seems like your latest internal version used that.

Reply:
i was only using that to ensure not setting the value worked. I presume we should keep it short so just the `@third_party//foo/bar` ?